### PR TITLE
Add svc_rest simple invocation examples to postman

### PR DESCRIPTION
**Pull request by miklosduma**
Have a look at the [built docs](https://galactic-robot-3035.postman.co/collections/9215924-be7fcbdd-586e-4fdb-9418-7df7b87393fd?version=latest&workspace=a8204d81-221a-41cf-8783-2fb0cb0274eb).

### CODE REVIEW

File: apps/sse_yaws/priv/docroot/postman/SPARKL.json [hunk 1]
```diff
@@ -2375,6 +2375,203 @@
...
+								}
+							]
+						},
+						"description": "Use it to trigger a notify or solicit on a REST client service. The fields and their value to be sent can be specified as query parameters."
+					},
+					"response": [
+						{
+							"name": "Call solicit or notify using form-data",
+							"originalRequest": {
+								"method": "GET",
```
Comment (lucafavatellawork) [inline apps/sse_yaws/priv/docroot/postman/SPARKL.json:2526]: Related to preceding GET it seems.

File: apps/sse_yaws/priv/docroot/postman/SPARKL.json [hunk 2]
```diff
@@ -2375,6 +2375,203 @@
...
+							],
+							"cookie": [],
+							"body": "{\n    \"data\": {\n        \"some_json_field\": {\n            \"content\": \"to send\",\n            \"some\": \"json\"\n        }\n    },\n    \"response\": \"{{response}}\"\n}"
+						}
+					]
+				},
+				{
+					"name": "Call solicit or notify using query params - REST",
+					"request": {
+						"method": "GET",
```
Comment (lucafavatellawork) [inline apps/sse_yaws/priv/docroot/postman/SPARKL.json:2497]: Is it really GET? It looks strange, as it seems call may have side effects. Though I did not check code.

Comment (miklosduma) [reply apps/sse_yaws/priv/docroot/postman/SPARKL.json:2497]: Supposed to handle both GET and POST: https://github.com/opensparkl/sse_core/blob/master/apps/svc_rest/src/svc_rest_yapp.erl#L162

File: apps/sse_yaws/priv/docroot/postman/SPARKL.json (hunk 1 above)
Comment (miklosduma) [reply apps/sse_yaws/priv/docroot/postman/SPARKL.json:2526]: Nope, I renamed the request. Seems like the example response's name was not updated. Nice catch!

**Pull request merged**